### ISSUE TITLE
No documentation for config.installedESLint

### ISSUE THREAD

**Issue by DamienCassou** [eslint/eslint#7758]
When using `eslint --init`, the new `.eslintrc.js` contains:

```js
{
    installedESLint: true
}
```

This option is not documented anywhere. This is related to https://github.com/eslint/eslint/issues/6200.

**Comment by not-an-aardvark** [comment]
Thanks for the question. I'm also unsure of what the option does.

Looking at the codebase, it only appears in two places: [here](https://github.com/eslint/eslint/blob/7079c8930fff7225f49c59d0513a6c42e59fb2b4/lib/config/config-initializer.js#L50) and [here](https://github.com/eslint/eslint/blob/7079c8930fff7225f49c59d0513a6c42e59fb2b4/lib/config/config-initializer.js#L87). Both of these seem to be used as part of the config generation process, and so it appears that the option isn't actually used once the config is generated. If this is the case, then it looks like a bug in ESLint that the property appears in the generated config at all. Tentatively, I think it should be fine to simply remove the option from your config without changing anything.

It looks like this only started happening in https://github.com/eslint/eslint/pull/5873, so I'll ask @kaicataldo to weigh in in case the option is actually used for something else that I've missed.

**Comment by kaicataldo** [comment]
This seems like an unintended consequence of my change - I don't think I intended for that to end up in the actual config. We should fix this.

**Comment by captDaylight** [comment]
I was still experiencing this until I globally reinstalled eslint. Thanks!!

**Comment by himdel** [comment]
@captDaylight you may need to manually remove `"installedJsLint": true` from your eslintrc.

**Comment by kaicataldo** [comment]
I believe this was fixed more than a year ago. @himdel is correct, though - the fix was to not generate with the unused property, not to removed pre-existing instances of it. Sorry for the confusion!
